webapp/actions/channel_actions.jsx

```diff
@@ -76,6 +76,9 @@ export function executeCommand(message, args, success, error) {
     msg = cmd + msg.substring(cmdLength, msg.length);
 
     switch (cmd) {
+    case '/search':
+        GlobalActions.emitSearchEvent(msg.substring(cmdLength + 1, msg.length));
```

Comment:
We actually already have a function for this in `post_actions.jsx`. Could you switch this to use that one?